helix_fhir_client_sdk/fhir_delete_mixin.py

```diff
@@ -118,21 +113,31 @@ async def delete_by_query_async(
             retries=self._retry_count,
             exclude_status_codes_from_retry=self._exclude_status_codes_from_retry,
             use_data_streaming=self._use_data_streaming,
+            compress=False,
+            throw_exception_on_error=self._throw_exception_on_error,
         ) as client:
             response: RetryableAioHttpResponse = await client.delete(
-                url=full_uri.tostr(), headers=headers
+                url=full_url, headers=headers
             )
             request_id = response.response_headers.get("X-Request-ID", None)
             self._internal_logger.info(f"X-Request-ID={request_id}")
             if response.status == 200:
                 if self._logger:
                     self._logger.info(f"Successfully deleted: {full_uri}")
 
+            deleted_count: Optional[int] = None
+            response_text = await response.get_text_async()
+            if response_text and response_text.startswith("{"):
+                # '{"deleted":0}'
+                deleted_info = json.loads(response_text)
+                deleted_count = deleted_info.get("deleted", None)
```

Comment:
what are the deleted info and count for??

Reply:
returns the count of resources deleted by the FHIR server